docs/Interaction_Guidelines/ux_controls_ui_patterns.md

```diff
@@ -59,8 +59,9 @@ A button communicates what happens when a user touches it. Button labels can con
 ![buttons](imgs/3-buttons-cover.png)
 
 ### Usage
+
 | Button | How to use |
-| --- | --- |
+|----|:-----|
```

Comment:
Did we check with @d45 on the correct syntax for this one?

Reply:
Yes. I see the change as part of this commit: https://github.com/tableau/extensions-api/commit/af7697d25cfc8ed9a093f4836f2a94c7e641502a . @d45 can you please verify?

Reply:
@johnDance  Yes. This is what I have working locally (🤞 that it works on GitHub). The table on GitHub is currently broken.